hs_core/templates/pages/genericresource.html

```diff
@@ -208,12 +210,14 @@ <h4 class="modal-title">Confirm file deletion</h4>
                             {#  Resource specific metadata can't be edited for HIS Referenced Time Series #}
                             {% if cm|resource_type != "HIS Referenced Time Series" or not metadata_form %}
                                 {% if  cm|resource_type != "Generic" %}
-                                    <li class="col-xs-2 col-md-2">
-                                        <a href="#resourceSpecificTab" data-toggle="tab" title="Resource specific">
-                                            <span class="glyphicon glyphicon-folder-close"></span>
-                                            &nbspResource Specific
-                                        </a>
-                                    </li>
+                                    {% if  cm|resource_type|lower != "collection resource" %}
```

Comment:
@zhiyuli Just realized resource_type is a customized filter that returns verbose name for the resource, so the space in "collection resource" makes sense to me now, so you can disregard my second comment above regarding resource type check.